client_test.go

```diff
@@ -38,6 +38,53 @@ func TestGetMarathonURL(t *testing.T) {
 	assert.Equal(t, endpoint.Client.GetMarathonURL(), endpoint.URL)
 }
 
+func TestApiRequest(t *testing.T) {
+	cases := []struct {
+		Auth     bool
+		Username string
+		Password string
+		Ok       bool
+	}{
+		{
+			Username: "bad",
+			Ok:       true,
+		},
+		{
+			Auth:     true,
+			Username: "bad",
+		},
```

Comment:
I'd also define `Password` and `Ok` explicitly here in order to improve readability of the test.